statics/js/components/graph-layout.js

```diff
@@ -1094,13 +1100,37 @@ TopologyGraphLayout.prototype = {
                  'analyzer.bandwidth_absolute_alert']
      };
 
+    if (typeof(Storage) !== "undefind") {
+      var cfgValues = {
+      absolute: [this.getLocalValue("bandwidthAbsoluteActive"),
+                 this.getLocalValue("bandwidthAbsoluteWarning"),
+                 this.getLocalValue("bandwidthAbsoluteAlert")],
+      relative: [this.getLocalValue("bandwidthRelativeActive"),
+                 this.getLocalValue("bandwidthRelativeWarning"),
+                 this.getLocalValue("bandwidthRelativeAlert")]
+      };
+    } else {
+      var cfgValues = {
```

Comment:
can you move the declaration before the if so that we don't use var in if/else statement which is more readable

Reply:
done